edd-discount-widget.php

```diff
@@ -21,13 +21,23 @@
     /**
      * Main EDD_Discounts_Widget class
      *
-     * @since       1.0.0
+     * @since       1.0.3
      */
     class EDD_Discounts_Widget extends WP_Widget {
-        function edd_discounts_widget() {
-            $widget_ops = array( 'classname' => 'edd_discounts_widget', 'description' => __( 'Display current discounts from any EDD powered site!', 'edd-discounts-widget' ) );
-            $control_ops = array( 'id_base' => 'edd_discounts_widget' );
-            $this->WP_Widget( 'edd_discounts_widget', __( 'Easy Digital Downloads - Discounts', 'edd-discounts-widget' ), $widget_ops, $control_ops );
+        public function __construct() {
+            $widget_ops = array(
+                'classname'   => 'edd_discounts_widget',
+                'description' => __( 'Display current discounts from any EDD powered site!', 'edd-discounts-widget' )
+            );
+            $control_ops = array(
+                'id_base' => 'edd_discounts_widget'
+            );
+            parent::__construct(
+                'edd_discounts_widget',
+                __( 'Easy Digital Downloads - Discounts', 'edd-discounts-widget' ),
+                $widget_ops,
```

Comment:
This is good, but what do you think about inlining these arrays since they're only used in one place? So instead of assigning them to a variable, just put them in here directly.